file1

```diff
@@ -2,4 +2,5 @@ Hello from Adal76 on 13/10
 Hello from AndreasDal on 13/10
 Hello again from Adal76 on 13/10
 Hello again from AndreasDal on 13/10
```

Comment:
this line is really important!

Reply:
Ok, I see what you mean! I will have a closer look at it and get back.

Reply:
ohh, I see that you added a new line since I commented last! I have a look at it.

Reply:
I see your point and rest my case ..